src/__tests__/medications/search/MedicationRequestTable.test.tsx

```diff
@@ -1,39 +1,49 @@
-import { render, screen, waitFor } from '@testing-library/react'
+import { render as rtlRender, screen, waitFor } from '@testing-library/react'
```

Comment:
we are wanting to avoid aliasing render. 

utilize `setup` function as your `render()` wrapper